libs/common/src/helpers/amendment-future-effectiveDate.ts

```diff
@@ -0,0 +1,9 @@
+import { fromUnixTime } from 'date-fns';
+import { UnixTimestampSeconds } from '../types/date';
+import { now } from './date';
+/**
+ * Returns a boolean indicating whether the given amendment effective date is in the future.
+ *
+ * @returns {Boolean} If the effective date is in the future, returns true; otherwise, false.
```

Comment:
do not need the type here as is ts file
```suggestion
 * @returns If the effective date is in the future, returns true; otherwise, false.
```